src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,206 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String insertCarRequest = "INSERT INTO cars (manufacturer_id, model) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                   PreparedStatement insertCarStatement
+                           = connection.prepareStatement(insertCarRequest,
+                           Statement.RETURN_GENERATED_KEYS)) {
+            insertCarStatement.setLong(1, car.getManufacturer().getId());
+            insertCarStatement.setString(2, car.getModel());
+            insertCarStatement.executeUpdate();
+            ResultSet resultSet = insertCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert car to DB: " + car, e);
+        }
+        insertDriver(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectCarRequest = "SELECT cars.id AS car_id, name, model, country, "
+                + "manufacturers.id AS manufacturer_id FROM cars JOIN manufacturers ON"
+                + " cars.manufacturer_id = manufacturers.id WHERE cars.id = ?"
+                + " AND cars.is_deleted = FALSE;";
```

Comment:
Please try to split your queries in more tidy and readable way. Spaces are better in the end of the row. Rows should be logically split to have full logical command in single row:
```suggestion
        String selectCarRequest = "SELECT cars.id AS car_id, name, model, country, "
                + "manufacturers.id AS manufacturer_id "
                + "FROM cars " 
                + "JOIN manufacturers ON cars.manufacturer_id = manufacturers.id "
                + "WHERE cars.id = ? "
                + "AND cars.is_deleted = FALSE;";
```